src/PanelTraits/Update.php

```diff
@@ -38,26 +40,25 @@ public function update($id, $data)
     /**
      * Get all fields needed for the EDIT ENTRY form.
      *
-     * @param  [integer] The id of the entry that is being edited.
-     * @param int $id
+     * @param int $id The id of the entry that is being edited.
      *
-     * @return [array] The fields with attributes, fake attributes and values.
+     * @return array The fields with attributes, fake attributes and values
      */
     public function getUpdateFields($id)
     {
         $fields = $this->update_fields;
         $entry = $this->getEntry($id);
 
-        foreach ($fields as $k => $field) {
+        foreach ($fields as &$field) {
```

Comment:
Cause it introduces some purpose of "immutability". Instead of doing you can create another array of the property and merge it, it will be more "readable" and cleaner